azext_iot/central/models/devicetwin.py

```diff
@@ -11,6 +11,11 @@ def __init__(
         device_twin: dict,
     ):
         self.device_twin = device_twin
+        if "_links" in device_twin:
+            device_twin.pop("_links")
+        if "tags" in device_twin:
+            device_twin.pop("tags")
```

Comment:
Why pop device twin tags? Tags are a [first class](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-device-twins#device-twins) concept for device twins.

Reply:
Second - tags are needed for digital twins

Reply:
> Second - tags are needed for digital twins

Device twins not digital twins. Still something we would want to keep

Reply:
current "twin show" command does not output tags. I just wanted to keep it consistent. But since it's an addition and won't break anything I refactor